src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesEdit/ExternalDataSourcesEdit.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import { renderWithIntl, TestForm } from '@folio/stripes-erm-testing';
+import { Button, Select, TextArea, TextField, Checkbox } from '@folio/stripes-testing';
+import translationsProperties from '../../../../../test/helpers';
+
+import { input, meta, initialValues } from './testResources';
+import ExternalDataSourcesEdit from './ExternalDataSourcesEdit';
+
+const onCancelMock = jest.fn();
+const onSaveMock = jest.fn();
+const onSubmit = jest.fn();
+
+let renderComponent;
+
+describe('ExternalDataSourcesEdit', () => {
+  beforeEach(() => {
+    renderComponent = renderWithIntl(
+      <TestForm
+        initialValues={initialValues}
+        onSubmit={onSubmit}
+      >
+        <ExternalDataSourcesEdit
+          input={input}
```

Comment:
We should never send `input` or `meta` props in directly unless there's a very good reason to.